pkgs/development/interpreters/python/hooks/pip-build-hook.sh

```diff
@@ -23,8 +23,12 @@ pipShellHook() {
       tmp_path=$(mktemp -d)
       export PATH="$tmp_path/bin:$PATH"
       export PYTHONPATH="$tmp_path/@pythonSitePackages@:$PYTHONPATH"
+      if [ -z "$enableParallelBuilding" ]; then
+        export MAKEFLAGS+=" -j$NIX_BUILD_CORES -l$NIX_BUILD_CORES"
+      fi
       mkdir -p "$tmp_path/@pythonSitePackages@"
       @pythonInterpreter@ -m pip install -e . --prefix "$tmp_path" \
+         ${enableParallelBuilding:+--install-option="--parallel=$NIX_BUILD_CORES"} \
```

Comment:
this option is also build-system dependent, in this case, setuptools. It is for `setup.py install` which is not used by this hook, as this hook installs a wheel that was built in the build phase.

Reply:
Removed it